app/main.py

```diff
@@ -1,6 +1,34 @@
+import requests
+from pprint import pprint
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
```

Comment:
According to the checklist, you should use constants. I see you are using a constant for the URL, which is good. However, you should also consider defining a constant for the city name, instead of fetching it directly from the environment variables. This way, you can easily change the city name in one place, if needed.